dataflows/processors/dumpers/file_formats.py

```diff
@@ -203,3 +201,41 @@ def write_transformed_row(self, transformed_row):
 
     def finalize_file(self):
         self.writer.write(']')
+
+
+class GeoJSONFormat(JSONFormat):
+    def __init__(self, file, schema, **options):
+        writer = file
+        writer.write('{"type": "FeatureCollection","features":')
+        writer.__first = True
+        super(GeoJSONFormat, self).__init__(writer, schema, **options)
+
+    def write_transformed_row(self, transformed_row):
+        if not self.writer.__first:
+            self.writer.write(',')
+        else:
+            self.writer.__first = False
+        try:
+            geometry = {}
+            for k, v in transformed_row.items():
+                if self.fields[k].type == "geopoint":
+                    geometry = {"type": "Point",
+                                "coordinates" : self._FileFormat__transform_value(v, self.fields[k])}
```

Comment:
why not use `v` here instead of `self._FileFormat__transform_value(v, self.fields[k])`?